src/Type/InterfaceType/UniformResourceIdentifiable.php

```diff
@@ -57,8 +56,14 @@ public static function register_type( TypeRegistry $type_registry ) {
 							$type = null;
 					}
 
-					return $type;
-
+					/**
+					 * Filter the UniformResourceIdentifiable node type.
+					 *
+					 * @param null|string $type The type definition the node should resolve to.
+					 * @param object $node The $node that is being resolved.
+					 * @param TypeRegistry $type_registry The type registry.
+					 */
+					return apply_filters( 'graphql_resolve_uniform_resource_identifiable_node_type', $type, $node, $type_registry );
```

Comment:
@renatonascalves can you explain this filter a bit more? 

Since we're registering an InterfaceType here, this already gets passed through this filter here: https://github.com/wp-graphql/wp-graphql/blob/develop/src/Type/WPInterfaceType.php#L58

I'd prefer to keep filters more central like this, vs introducing new filters for every Type that's registered.

Reply:
Thanks for the link. Didn't know about that one. This is unnecessary. Will remove it. :)